boards/sltb001a/board.c

```diff
@@ -21,9 +21,8 @@
 
 #include "board.h"
 #include "board_common.h"
-#include "pic.h"
-
 #include "periph/gpio.h"
+#include "pic.h"
```

Comment:
Maybe this include could be protected with a `#ifdef MODULE_SILABS_PIC` ?

Reply:
I can add that, but it compiles fine without (if I remove `silabs_pic` from the `USEMODULES`).

Let me know what you prefer.

Reply:
On second thought: I'll add it. That's more similar to how we also include optional peripherals.